res/controllers/Korg-KAOSS-DJ-scripts.js

```diff
@@ -1,116 +1,147 @@
 // Korg KAOSS DJ controller mapping for Mixxx
-// Seb Dooris, Fayaaz Ahmed, Lee Arromba
-
-var KAOSSDJ = {};
-var ON = 0x7F,
-    OFF = 0x00,
-    UP = 0x01,
-    DOWN = 0x7F;
-var ledChannel = {
-    'btnsL': 0x97,
-    'btnsR': 0x98,
-    'knobsL': 0xB7,
-    'knobsR': 0xB8,
-    'master': 0xB6
-};
-var led = {
-    'cue': 0x1E,
-    'sync': 0x1D,
-    'play': 0x1B,
-    'headphones': 0x19,
-    'fx': 0x18, // warning: led is owned by controller
-    'stripL': 0x15,
-    'stripM': 0x16,
-    'stripR': 0x17,
-    'loopStripL': 0x0F,
-    'loopStripM': 0x10,
-    'loopStripR': 0x11,
-};
-
-// initialise decks
+// Seb Dooris, Fayaaz Ahmed, Lee Arromba, Raphael Quast
+
+// Using `const`/`let` causes ReferenceError: KAOSSDJ is not defined
```

Comment:
@Swiftb0y But why?

Reply:
We expect the "controller namespace" (`KAOSSDJ` in this case) to be accessible via the global object. That only happens when variable is declared via `var` in the global context. Declaring it using `const` or `let` won't do that. You can confirm this in node js using the following snippet:

```javascript
const a = "a";
let b = "b";
var c = "c";

console.log(`a: ${a}, global.a: ${global.a}\nb: ${b}, global.b: ${global.b}\nc: ${c}, global.c: ${global.c}`);
```

I'm not sure if we can access these variables without the global object, but we could try.

Reply:
Fix shouldn't be particularly difficult, I expect about 10-20 extra lines. Though I don't have time to look into this right now.